indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -151,7 +155,9 @@ public IndexTask(
       @JsonProperty("resource") TaskResource taskResource,
       @JsonProperty("spec") IndexIngestionSpec ingestionSchema,
       @JacksonInject ObjectMapper jsonMapper,
-      @JsonProperty("context") Map<String, Object> context
+      @JsonProperty("context") Map<String, Object> context,
+      @JacksonInject IndexMerger indexMerger,
+      @JacksonInject IndexIO indexIO
```

Comment:
@gianm my bad, missed the task serde test. In general I think it would be nice to move away from jackson injecting if we ever want to make those classes available in a proper API or client library.